libdigidocpp.dox

```diff
@@ -403,6 +403,8 @@ http://www.w3.org/TR/2008/REC-xmldsig-core-20080610/xmldsig-core-schema.xsd
 </td></tr><tr><td>XAdES01903v132-201601.xsd</td><td>XML Advanced Electronic Signatures (\ref XAdES_EN "XAdES EN") 
 Defines a set of extensions to XML-DSIG making it suitable for advanced electronic signature.
 http://uri.etsi.org/01903/v1.3.2/XAdES01903v132-201601.xsd
+</tr><tr><td>XAdES01903v132-201601-relaxed.xsd</td><td>Relaxed version of XML Advanced Electronic Signatures (\ref XAdES_EN "XAdES EN")
+Allows to list child elements of the SignatureProductionPlace element in any sequence. Schema is used when validating the BDOC documents in order to accept documents produced by DigiDoc4j library.
```

Comment:
Please update the comment to:
... by DigiDoc4J library version 1.0.3 and earlier.